libnavigation-core/src/main/java/com/mapbox/navigation/core/adasis/AdasisConfigCycleTimes.kt

```diff
@@ -0,0 +1,64 @@
+package com.mapbox.navigation.core.adasis
+
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+
+/**
+ * Message generation cycle times configuration
+ *
+ * @param metadataCycleOnStartMs time in milliseconds between sending metadata message on start
+ * @param metadataCycleSeconds time in seconds between repetition of metadata message
+ * @param positionCycleMs time in milliseconds between sending position message
+ */
+@ExperimentalPreviewMapboxNavigationAPI
+class AdasisConfigCycleTimes(
```

Comment:
Native object also has an empty constructor with default parameters. Don't we need it too?